recipes/graylog-forwarder/files/graylog-forwarder.service

```diff
@@ -16,6 +16,7 @@ WorkingDirectory=/var/lib/graylog-forwarder
 ExecStart=/usr/share/graylog-forwarder/bin/graylog-forwarder run -f /etc/graylog/forwarder/forwarder.conf
 
 Environment="FORWARDER_JVM_OPTIONS_FILE=/etc/graylog/forwarder/jvm.options"
+Environment="JAVA_OPTS=-Dlog4j.configurationFile=file:/etc/graylog/forwarder/log4j2.xml"
```

Comment:
Does `file:/etc/...` actually work? It's not a valid file URL. In the server we use: `-Dlog4j.configurationFile=file:///etc/graylog/server/log4j2.xml`

Reply:
Yeah, it works. I tested it in a VM. I'll put the extra slashes back in, though, since that seems to work, too. That was just from me trying to get it to work during the troubleshooting process.

Reply:
Oh neat, check it out:

```
The // after the file: denotes that either a hostname or the literal term localhost will follow,[2] although this part may be omitted entirely, or may contain an empty hostname.[3]
The single slash between host and path denotes the start of the local-path part of the URI and must be present.[4]
A valid file URI must therefore begin with either file:/path, file:///path or file://hostname/path.
```

Reply:
Ah, nice! :+1: Thanks for the update!